### ISSUE TITLE
Proposal: Re-do the structure?

### ISSUE THREAD

**Issue by chicken-biryani** [conda-forge/conda-forge.github.io#1672]
### Where should the content be added?

Modification in the current docs

### What should be added?

Currently, all the possible sections are intertwined with each other. For example, in staging recipes, I have to jump around a lot to be able to contribute.

Instead of keeping the structure as User documentation, Maintaining, Organization..

We make it section-wise as:

- Getting started 
     - Introduction to conda forge.
     -  Installation.
     - Glossary/Concepts Keeping newcomers in mind
     - Contributing-- Ways of contributing:
          - Infrastructure (About infrastructure, A brief on the repos related to infrastructure and redirection to them)
          - Packages (About, Repos involved)
           ....
           
* Expanding on Contributing to * (Just an example, I don't hold a lot of knowledge regarding the possible sections)
- Staging recipes
     - Intro to staging recipe (What is it?)
     - Glossary (Optional, either here or towards the beginning)
    - How to contribute to Staging recipes/ Maintain

- Xyz
     - Intro
     - Glossary
     - How to contribute/Maintain
 ..........

### Additional information

The proposal is rough and just an idea. Would love to hear what others have in mind about the same.

PS: The process may be lengthy. But maybe worth it, as new repositories/concepts are created we won't have to look for a way to fit them in the documentation. We could create a section and note it down.

**Comment by chicken-biryani** [comment]
@beckermr @kathatherine @BastianZim @jaimergp

**Comment by jakirkham** [comment]
Why was this closed?

**Comment by chicken-biryani** [comment]
Hi, @jakirkham I think I made the suggestion without having enough knowledge.

After reading the docs a little more, I realised that it's ALMOST in the way I've suggested that is-- talking about infrastructure related topics under infrastructure. About the parts of packages under well packages etc.

Except, the introduction part which will be covered in an issue made by Surbhi right after this one... And the concepts part which I'll be covering.

Yes, the docs do need restructuring-- looking at the commit history I am assuming it was created 3 years ago, I am sure a lot has changed since then-- 

but THIS proposal just didn't seem to have the spark, it didn't feel right. May require more brainstorming.
